blog/2024-10-8-self-hosting-reflex-with-docker.md

```diff
@@ -0,0 +1,296 @@
+---
+author: Tom Gotsman
+date: 2024-10-8
```

Comment:
When checking the preview in nav it shows invalid date

Reply:
fixed